django_freeradius/base/admin_filters.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.admin import SimpleListFilter
+from django.utils.translation import ugettext_lazy as _
+
+
+class DuplicateListFilter(SimpleListFilter):
+    title = _('find duplicates')
+    parameter_name = 'duplicates'
+
+    def lookups(self, request, model_admin):
+        return (
+                ('username', _('username')),
+                ('value', _('value')),
+               )
```

Comment:
please keep the indentation style consistent with the rest of the codebase, eg:

```python
(('username', _('username')),
 ('value', _('value')),)
```

or:

```python
(
    ('username', _('username')),
    ('value', _('value')),
)
```